server/routes/reporting/standard-report/index.js

```diff
@@ -40,6 +40,9 @@
   module.exports = function(app) {
     standardReportRoutes(app, 'next-due');
     standardReportRoutes(app, 'undelivered');
+    require('../postponed-report')(app);
+    standardReportRoutes(app, 'postponed-pool');
```

Comment:
Thinking about this a bit more (and building my own :D) I would suggest we should have an arg on the `standardReportRoutes` function to say 'don't include the filter routes', otherwise they'll be there but (probably) be quite broken.

Maybe: 
``` 
   if (reportKeys[key].search) {
    app.get(`/reporting/${key}`,
      `${key}.filter.get`,
      auth.verify,
      standardFilterGet(app, key));

    app.post(`/reporting/${key}`,
      `${key}.filter.post`,
      auth.verify,
      standardFilterPost(app, key));
   }
```
 and then take the search key out of the reportKeys?

Reply:
Yeah that makes complete sense, have added this in, have  had to rejig a bit as report keys now requires 'app' to be passed in to build some urls that are needed to keep everything tidier in the controller.